### PR TITLE
654 my credits page pc m 5

### PR DESCRIPTION
**Pull request by PiranavanShanmugavadivelu**

### CODE REVIEW

**Review by gjermundgaraba — APPROVED** [review]

File: frontend/marketplace/src/components/CreditCard.vue [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          <p class="text-title32 font-bold">100.00 ABCD</p>
+          <div class="grid grid-cols-2 gap-3">
+            <button
+                class="btn rounded-sm text-greenPrimary font-bold text-title14 bg-buttonBlack normal-case border-none">
+              View certificate
+            </button>
+            <button
+                class="btn rounded-sm text-greenPrimary font-bold text-title14 bg-buttonBlack normal-case border-none">
+              Retire credits
+            </button>
```
Comment (jschill) [inline frontend/marketplace/src/components/CreditCard.vue:27]: Two buttons with the same style calls for a <Button /> 😄

**Review by jschill — CHANGES_REQUESTED** [review]

**Review by ba1ciu — CHANGES_REQUESTED** [review]
I think we should already build it dynamically - not hardcode values into HTML, like hardcoded pagination in `MyCredits.vue` or the data on `CreditCard.vue`, but instead read it from variables and iterate if necessary.

File: frontend/marketplace/src/components/NavBar.vue [hunk 2]
```diff
@@ -102,4 +102,10 @@
 
 </template>
 
+<style scoped>
+.nav-dropdown-button{
```
Comment (jschill) [inline frontend/marketplace/src/components/NavBar.vue:106]: Nice!

File: frontend/marketplace/src/components/NavBar.vue [hunk 3]
```diff
@@ -54,14 +54,14 @@
                 </div>
 
                 <div class="menu py-2 items-center w-full">
-                  <button
-                      class="btn btn-ghost rounded-sm text-greenPrimary font-bold text-title14 normal-case border-none">
+                  <Button
```
Comment (jschill) [inline frontend/marketplace/src/components/NavBar.vue:57]: There is no Burton or is it registered globally?

**Pull request merged**